contrib/IECoreArnold/include/IECoreArnold/NodeAlgo.h

```diff
@@ -90,6 +90,11 @@ class ConverterDescription
 
 };
 
+/// Arnold does not support non-uniform sampling.  It just takes a start and end time, and assume
+/// the samples are distributed evenly between them.  We need to throw an exception if given data
+/// we can't render.
+void ensureUniformTimeSamples( const std::vector<float> &times );
```

Comment:
Can we just make this internal to the Renderer class? I'm trying to minimise the API surface area...

Reply:
I currently use this from both IECoreArnold::RendererImplementation and GafferArnold::IECoreArnoldPreview::Renderer, so I need it to be public.  I can add a /todo to make it private in Cortex 10 when we don't have this split.